op-program/host/host.go

```diff
@@ -80,6 +80,9 @@ func makeDefaultPrefetcher(ctx context.Context, logger log.Logger, kv kvstore.KV
 	if !cfg.FetchingEnabled() {
 		return nil, nil
 	}
+	if cfg.L1BeaconSkipBlobVerification {
+		logger.Warn("Blob verification is DISABLED. This should only be used for testing and development. NOT FOR PRODUCTION USE.")
```

Comment:
Maybe too intense. Will change based on whatever we use here: https://github.com/ethereum-optimism/optimism/pull/17713/files#r2403205232